### PR TITLE
Transaction result conversion

### PR DESCRIPTION
**Pull request by bachdavi**
This PR adds filtering for the transaction results, such that only Arrow encoded parts of the multi-part are deserialized. Using this I cleaned up the fast-path response and added a test exercising it. This uses a recent fix in HTTP.jl that correctly parses the `content-type`. @quinnj Are we already on `#master` of HTTP.jl? Otherwise should we do another release in HTTP.jl? :) 

The current RAI engines send a `relation-count` part in the `/transaction/results/` path that was breaking the result conversion.

@NRHelmi adding you as I don't know how to do a new release. Should we just create a new tag?

### CODE REVIEW

**Comment by bachdavi** [comment]
As the tests show we are not on `#master` of HTTP.jl :) 
Should we do another release in HTTP.jl @quinnj ?

**Comment by quinnj** [comment]
We're very close on HTTP.jl 1.0 release; my goal is in the next few (business) days. We've had a number of breaking changes, so it's a bit hard to parse out bugfixes along with the bigger breaking refactorings.

**Comment by NHDaly** [comment]
🤔 yeah not sure what to do about the HTTP 1.0 requirement.. 🤔 maybe we need to hardcode another crappy fix for this in the meantime, or just wait until it's released. :) Thanks for the fix, david!

**Review by NHDaly — APPROVED** [review]
✅ lgtm, thanks for the fix. 

Since this requires HTTP 1.0, can you also add an HTTP compat lower bound to `[compat]` in Project.toml? (It won't work until 1.0 is released, of course, but i guess that's coming soon! 🎉)

**Comment by NRHelmi** [comment]
@bachdavi I remember you need to comment or create an issue that contains
```
@JuliaRegistrator register
```
hopefully this won't trigger the package registration now :sweat_smile:

**Comment by NHDaly** [comment]
I think HTTP 1.0 has been released. Can we add the compat bounds and try this again? :)

**Comment by bachdavi** [comment]
> I think HTTP 1.0 has been released. Can we add the compat bounds and try this again? :)

I added `1.0` to compat here on this branch. Running the tests just now
